tests/inspectdb/tests.py

```diff
@@ -354,6 +355,25 @@ def test_table_name_introspection(self):
         output = out.getvalue()
         self.assertIn("class InspectdbSpecialTableName(models.Model):", output)
 
+    def test_custom_normalize_table_name(self):
+        def pascal_case_table_only(table_name):
+            return table_name.startswith("inspectdb_pascal")
+
+        class MyCommand(inspectdb.Command):
+            def normalize_table_name(self, table_name):
+                normalized_name = table_name.split(".")[1]
+                if connection.features.ignores_table_name_case:
```

Comment:
I'd think `normalize_table_name()` should take `connection` parameter rather than using (possibly incorrect) default connection. Or possibly should `normalize_table_name()` (also?) be a `DatabaseIntrospection` method?

Reply:
`normalize_col_name()` doesn't use any db-specific tweaks, so I assumed there was no need to complicate `normalize_table_name()` :thinking:

Reply:
I thought `if connection.features.ignores_table_name_case:` might be demonstrating a use case.

Reply:
> I thought `if connection.features.ignores_table_name_case:` might be demonstrating a use case.

Agreed, we could add `connection` to both methods.